docs/index.rst

```diff
@@ -0,0 +1,19 @@
+Welcome to Smorest CRUD's documentation!
+============================================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   source/README.md
+
```

Comment:
this file could be better
it renders this: https://smorest-crud.readthedocs.io/en/doc/